README.md

```diff
@@ -63,27 +63,4 @@ After the above steps your project structure should look like this:
     └── project-file-2
 
 1 directory, 4 files
-```
-
-
-## Usage
-
-The below steps describe how to use the hook on a sample repo provided by Private AI.
-
-1. Fork the repo located at `https://github.com/privateai/pai-pre-commit-demo.git`. The hook is already installed and configured to detection PII using Private AI's cloud endpoint
-2. Add your API_KEY as per step 6 in the Installation instructions
-2. The repo contains a file `sample_code.py`, which contains some PII mentions. Let's go ahead and modify it as mentioned in the comments
-3. Run `git add sample_code.py` to add it to the next commit
-4. Run `git commit -m "test"`
-5. If the hook is installed and configured correctly, the commit will fail with an error message similar to:\
-    `PII found - type: NAME_GIVEN, line number: 2, file: sample_code.py, start index: 11, end index: 17` \
-    `PII found - type: NAME_GIVEN, line number: 2, file: sample_code.py, start index: 21, end index: 24` \
-    `PII found - type: NAME_GIVEN, line number: 2, file: sample_code.py, start index: 28, end index: 34` \
-    `PII found - type: AGE, line number: 4, file: sample_code.py, start index: 9, end index: 11` \
-    `PII found - type: AGE, line number: 4, file: sample_code.py, start index: 13, end index: 15` \
-    `PII found - type: AGE, line number: 4, file: sample_code.py, start index: 17, end index: 19` 
```

Comment:
I am not clear if we had consensus on this. I agree with this change but I think we discussed creating a separate ticket at the dev meeting.

Reply:
Oh yes. @yaysummeriscoming Since it needs to be done anyway, should I go ahead with this change here or address this in a separate ticket?

Reply:
Sorry for my late reply. Separate ticket please